source/extensions/filters/http/jwt_authn/extractor.h

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <map>
+#include <string>
+#include <unordered_set>
+
+#include "envoy/config/filter/http/jwt_authn/v2alpha/config.pb.h"
+
+#include "common/common/logger.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+/**
+ * JwtLocation stores following info
+ * *  extracted JWT string,
+ * *  the location where the JWT is extracted from,
+ * *  list of issuers specified the location. The issuer of extracted JWT must
```

Comment:
Nit: can you reflow this so that the wrap is still indented?

Reply:
Done